web/templates/pages/members.html

```diff
@@ -19,14 +19,14 @@
 		<div>
 			<ul class="about_list">
 			{% for member in members_list %}
-				{% if member.activity_status == 'MC' %}
-					<li class="mentor-cat"><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Mentor cat)<li>
-				{% elif member.activity_status == 'AC' %}
-					<li class="active-cat"><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Active cat)<li>
-				{% elif member.activity_status == 'LC' %}
-					<li class="lazy-cat"><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Lazy cat)<li>	
-				{% elif member.activity_status == 'SC' %}
-					<li class="sleepy-cat"><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Sleepy cat)<li>
+				{% if member.activity_status == 'MENTORCAT' %}
+					<li class="mentor-cat"><a href="{% url 'web.views.member_page' user_id=member.id %}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Mentor cat)<li>
```

Comment:
My previous comment on urls.py applies here. You failed to name the URL and now you are using this very ugly reference to the view.

This line(s) are also too long and are not nice to read.
https://www.python.org/dev/peps/pep-0008/#maximum-line-length